.husky/pre-push

```diff
@@ -0,0 +1 @@
+npm run lint && npm test
```

Comment:
I would remove test, since sometimes you want to push when tests are wip

Reply:
we should `.skip` tests you want to push that are still in progress.

the unit tests in SDK are quick to run (unlike smart-contracts) so it could be worth adding to pre-push hook.

but if you feel strongly we should not have it, please let me know and I'll drop it.